Sources/Foundation/FileManager+POSIX.swift

```diff
@@ -1315,7 +1329,7 @@ extension FileManager {
             let finalErrno = originalItemURL.withUnsafeFileSystemRepresentation { (originalFS) -> Int32? in
                 return newItemURL.withUnsafeFileSystemRepresentation { (newItemFS) -> Int32? in
                     // This is an atomic operation in many OSes, but is not guaranteed to be atomic by the standard.
-                    if rename(newItemFS, originalFS) == 0 {
+                    if let newFS = newItemFS, let origFS = originalFS, rename(newFS, origFS) == 0 {
```

Comment:
Why not simply force unwrap?

Reply:
That's better, dropped this in #5010, you can do that in a version of #4889 for 6.0.